labelbox/schema/data_row_metadata.py

```diff
@@ -172,6 +175,21 @@ def get_by_name(
         else:
             raise KeyError(f"There is no metadata with name {name}")
 
+    def _get_schema_by_name(
```

Comment:
Curious, why not use `self.get_by_name(name)` here? Looks like a bit of duplication and I wonder if we need to consider using this method here or creating a new with better naming with explicitly says "get_by_normalized_name" or similar

Reply:
For enums reseved_by_name and custom_by_name are dicts, they don't provide the id of the field, so I can't use `get_by_name`. I agree the duplication here is not ideal, I am open to ideas. When reading through the code, I personally found the term `normalized` a little confusing and wanted to avoid using that in the function.

Reply:
I have extracted a common lookup function used by both functions and renamed the new function to `_ get_by_name_normalized`. `nested` and `flat` index might be better terminology, but don't want to change too much in this PR.